src/Npgsql/NpgsqlReadBuffer.cs

```diff
@@ -31,29 +31,29 @@ public sealed partial class NpgsqlReadBuffer : IDisposable
 
         readonly Socket? _underlyingSocket;
 
-        readonly TimeoutCancellationTokenSourceWrapper _timeoutCts;
+        internal TimeoutCancellationTokenSourceWrapper TimeoutCts { get; }
```

Comment:
nit: technically, this isn't just about timeouts - it's just the CancellationTokenSource for the buffer, since we also cancel it directly :trollface:

(but I'm fine with the naming, doesn't matter)

Reply:
🍞